plugins/repository-s3/src/test/java/org/elasticsearch/repositories/s3/AmazonS3Fixture.java

```diff
@@ -246,9 +253,18 @@ public static void main(final String[] args) throws Exception {
                         destBucket.objects.put(destObjectName, bytes);
                         return new Response(RestStatus.OK.getStatus(), TEXT_PLAIN_CONTENT_TYPE, EMPTY_BYTE);
                     }
-                }
+                } else {
+                    if (!disableChunkedEncoding) {
```

Comment:
Style nit:

```suggestion
                    if (disableChunkedEncoding == false) {
```

Experience has shown that a vital `!` is too easy to miss, so we prefer `== false` instead.